app/deliver_grant_funding/templates/deliver_grant_funding/developers/edit_question.html

```diff
@@ -1,54 +1,48 @@
-{% extends "platform/manage_grant_base.html" %}
+{% extends "deliver_grant_funding/manage_grant_base.html" %}
 {% from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary %}
 {% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
 {% from "govuk_frontend_jinja/components/inset-text/macro.html" import govukInsetText %}
 {% from "govuk_frontend_jinja/components/summary-list/macro.html" import govukSummaryList %}
 
 {% block pageTitle %}
-  Edit question - {{ db_form.title }} - MHCLG Funding Service
+    Edit question - {{ db_form.title }} - MHCLG Funding Service
 {% endblock pageTitle %}
 
 {% set active_sub_navigation_tab = "grant_developers" %}
 
 {% block beforeContent %}
-  {{
-    govukBackLink({
+    {{ govukBackLink({
         "text": "Back",
         "href": url_for("platform.manage_form", grant_id = grant.id, collection_id = collection.id, section_id=section.id, form_id=db_form.id, back_link="manage_section")
-    })
-  }}
+    }) }}
 {% endblock beforeContent %}
 
 {% block content %}
-  <h1 class="govuk-heading-l">Edit question</h1>
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-full">
-      {{
-        govukSummaryList({
+    <h1 class="govuk-heading-l">Edit question</h1>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            {{ govukSummaryList({
           "rows": [{
               "key": {"text": "Question type"},
               "value":{"text": question.data_type},
         }]
-        })
-      }}
-      {{
-        govukInsetText({
+        }) }}
+            {{ govukInsetText({
          "text": "You cannot change the type of a question once it has been created. If you need to change the type of a question, you will need to delete the question and create a new one.",
-        })
-      }}
+        }) }}
+        </div>
     </div>
-  </div>
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-full">
-      <form method="post" novalidate>
-        {{ form.csrf_token }}
-        {{ form.form_id }}
-        {{ form.question_data_type }}
-        {{ form.text }}
-        {{ form.name }}
-        {{ form.hint }}
-        {{ form.submit }}
-      </form>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            <form method="post" novalidate>
+                {{ form.csrf_token }}
+                {{ form.form_id }}
+                {{ form.question_data_type }}
+                {{ form.text }}
+                {{ form.name }}
+                {{ form.hint }}
+                {{ form.submit }}
+            </form>
+        </div>
```

Comment:
Something in your IDE blatting formatting in these files maybe?

Reply:
Seems to be. Have rerun Steven's formatter - might see if I can integrate that with something somehow this afternoon, and see what my IDE is doing 👀